articles/what-is-dns.mdx

```diff
@@ -0,0 +1,103 @@
+
+All nameservers have the store identical DNS records for a Domain Name. Updating the DNS records for the primary nameserver will trigger an update in the secondary server.
+
+## What are DNS records?
+
+DNS records are instructions for a Domain Name, managed by an authoritative nameserver. These instructions contain important information including a domain's IP address, email routing, owner validation, and more.
+
+Each DNS record has a Time-To-Live (TTL) value, which indicates how often a DNS server should refresh a record to check for updates from the domain owner.
+
+<comp.DocsImage path="https://cc-dam.imgix.net/documentation/images/hosting/configure-cloudcannon-dns/2024-11/CloudCannon-Documentation-Domain-Settings-CloudCannon-DNS.png" alt="A screenshot of the Domain Settings page shows the DNS Provider dropdown is set to CloudCannon DNS." title="The Domain Settings page" type="screenshot"/>
+
+CloudCannon DNS supports the following DNS record types: *A*, *AAAA*, *CNAME*, *MX*, *SPF*, *SRV*, and *TXT*.
+
+* **Address (A) Record** — Specifies the IP address for a given Domain Name in IPv4 format (e.g., 93.184.216.34). This record allows you to access a website if you only know the Domain Name, or block mail from a specific source.
+* **Address (AAAA) Record** — Specifies the IP address for a given Domain Name in IPv6 format (e.g., 3001:0db7:3c5d:0024:0000:0000:1a2f:3c1b). IPv6 is a newer format of IPv4, offering a longer string and more string combinations to increase the number of unique IP addresses available.
+* **Canonical Name (CNAME) Record** — Specifies the canonical Domain Name for an alias domain. Often used for subdomains (e.g., users looking for the IP address for www.example.com will be redirected to example.com) or for hosting platforms.
+* **Email Exchange (MX) Record** — Specifies where emails for a given Domain Name should be routed to.
+* **Sender Policy Framework (SPF) Record** — Specifies which servers are authorized to send emails from a given Domain Name.
+* **Service (SRV) Record** — Specifies the host and port number for services the need to connect to a specific port.
+* **Text (TXT) Record** — Can store any text string, but is commonly used to verify ownership of a Domain Name.
+
+Other record types include:
+
+* **Nameserver (NS) Record** — Specifies which nameserver is authoritative for a given Domain Name.
+
+Instead of configuring NS records in CloudCannon, you update your nameservers with your Domain Registrar. For more information, please read our documentation on [configuring CloudCannon DNS](/documentation/articles/configure-cloudcannon-dns/).
+
+## DNS types
+
+On CloudCannon, any Site with a [Custom Domain](/documentation/articles/what-is-a-custom-domain/) can use [CloudCannon DNS](/documentation/articles/configure-cloudcannon-dns/) or an [external DNS](/documentation/articles/configure-external-dns/).
```

Comment:
CloudCannon DNS (or any DNS provider) is not associated to a Site, but with a Domain. After you add the first Custom Domain to a site is when we trigger the creation of it all, but the DNS doesn't have any dependency on the site. 
It's currently confusing, but would be good to have that distinction. `configure-cloudcannon-dns` page also has a similar reference for a DNS to a Site that would need to be reviewed.